teradatalabs/cdh5-hive-kerberized/Dockerfile

```diff
@@ -69,6 +69,9 @@ ADD files/conf/hive-site.xml /etc/hive/conf/hive-site.xml
 RUN chown hive:hadoop /etc/hive/conf/hive.keytab
 RUN chmod 644 /etc/hive/conf/hive.keytab
 
+# ENABLE AUTHORIZATION IN HIVE SERVER 
+ADD files/conf/hiveserver2-site.xml /etc/hive/conf/hiveserver2-site.xml
```

Comment:
This is a standard file name for specifying HiveServer2 specific configuration values. I don't think we given a custom name to it.